.tree-sitter-lint/local_rules/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "local_rules"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+tree-sitter-lint = { path = "../.." }
```

Comment:
This would end up being a version dependency (where we generate it to match the version of `tree-sitter-lint` that's doing the code-generation), but for now just a "hard-coded" local path dependency